<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pbCorrect</class>
 <widget class="QMainWindow" name="pbCorrect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BASIL BCI: Steady state visually evoked potentials</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="pbConfidence">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblConfidence">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classifier confidence:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEEG">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>580</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EEG:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>600</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Markers:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEEGStatus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>580</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMarkerStatus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>600</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red</string>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbLSLStatus">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>560</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>LSL stream availability</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbStatusInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>791</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Status information</string>
    </property>
    <widget class="QTextEdit" name="teStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>771</width>
       <height>181</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbImageGuessed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Classification results:</string>
    </property>
    <widget class="QLabel" name="lblImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="gvPlots">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>651</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpectrum">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spectra from EEG channel:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbChannelID">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAllChannels">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use all channels</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbAppControl">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>640</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="pbStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbFeedback">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>730</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Optional feedback</string>
    </property>
    <widget class="QPushButton" name="pbTrue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>True</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>correct.png</normaloff>correct.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbFalse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>False</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>wrong.png</normaloff>wrong.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
